### ISSUE TITLE
Remove buildtools and all related configuration. [phab:D289]

### ISSUE THREAD

**Issue by kythe** [google/kythe#508]
_Created by **schroederc** at 2015-05-27 18:07:41_:

**Comment by kythe** [comment]
_**zarko** accepted this revision at 2015-05-27 18:15:14_

**Comment by kythe** [comment]
_**fromberger** accepted this revision at 2015-05-27 18:18:11_

**Comment by kythe** [comment]
_**schroederc** wrote at 2015-05-27 18:32:27_:

There's a second part to this change where we delete all of the unused third_party dependencies.  However, that commit is too large for Phabricator, and I will submit it without review after this change (if there are no complaints).

**Comment by kythe** [comment]
_**schroederc** wrote at 2015-05-27 18:37:50_:

These are the third_party directories to be removed:

```
third_party/gflags
third_party/glog
third_party/jcommander
third_party/ninja
third_party/node
third_party/protobuf
```

**Comment by kythe** [comment]
_**schroederc** closed via commit b8039a54381b03d81945b3a4c7e7ad0ada1b7e88 at 2015-05-27 20:59:02_
